web/src/main/resources/application.yaml

```diff
@@ -1,3 +1,5 @@
 spring:
   profiles:
-    active: dev
\ No newline at end of file
+    active: dev
+  main:
+    allow-circular-references: true
```

Comment:
@YauhenIlyin wtf???

Reply:
have not yet found another solution, it is necessary to show the architecture